From 3f66bc7c54c0e1de07efccebaa84d99f2f0238f1 Mon Sep 17 00:00:00 2001
From: Alberto Murillo Silva <alberto.murillo.silva@intel.com>
Date: Tue, 1 Nov 2016 15:10:45 -0600
Subject: [PATCH 1/2] Add make install target

Signed-off-by: Alberto Murillo Silva <alberto.murillo.silva@intel.com>
---
 Makefile | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Makefile b/Makefile
index 9d0f49c..f643b3f 100644
--- a/Makefile
+++ b/Makefile
@@ -107,3 +107,9 @@ tests/prlimit-nofile: tests/prlimit-nofile.c
 
 test: tests/__build-tstamp__ tests/prlimit-nofile
 	cd tests && env $(TEST_ENV) ./run
+
+install:
+	install -d -m 755 $(DESTDIR)/usr/bin
+	install -m 755 redsocks $(DESTDIR)/usr/bin
+	install -d -m 755 $(DESTDIR)/usr/lib/systemd/system
+	install -m 755 redsocks.service $(DESTDIR)/usr/lib/systemd/system
-- 
2.7.4

