From f6c87c485b9aa6632249a2090b245cd1686576d8 Mon Sep 17 00:00:00 2001
From: Alberto Murillo Silva <alberto.murillo.silva@intel.com>
Date: Tue, 1 Nov 2016 15:10:45 -0600
Subject: [PATCH] Add make install target

Signed-off-by: Alberto Murillo Silva <alberto.murillo.silva@intel.com>
---
 Makefile | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Makefile b/Makefile
index 5ab12cb..506adc7 100644
--- a/Makefile
+++ b/Makefile
@@ -109,3 +109,9 @@ tests/prlimit-nofile: tests/prlimit-nofile.c
 
 test: tests/__build-tstamp__ tests/prlimit-nofile
 	cd tests && env $(TEST_ENV) ./run
+
+install:
+	install -d -m 755 $(DESTDIR)/usr/bin
+	install -m 755 redsocks $(DESTDIR)/usr/bin
+	install -d -m 755 $(DESTDIR)/usr/lib/systemd/system
+	install -m 644 redsocks.service $(DESTDIR)/usr/lib/systemd/system
-- 
2.20.1

